fhir-validation/src/main/java/com/ibm/fhir/validation/util/ConstraintGenerator.java

```diff
@@ -39,6 +39,8 @@
  * A class used to generate FHIRPath expressions from a profile
  */
 public class ConstraintGenerator {
+    public static boolean DEBUG = false;
```

Comment:
can we use proper logging instead?

Reply:
Changed the class to use java.util.logging instead of global DEBUG flag.